lumps/textures/textures.cfg

```diff
@@ -1406,6 +1406,16 @@ TEKGREN4        64      128
 *   RW37_4      0       0
 TEKGREN5        64      128
 *   RW39_1      0       0
+TEKGREN6        64      128
+*   RW37_5      0       0
+TEKGRENB        64      128 ; Big
+*   RW37_6      0       0
+TEKGRENT        64      128 ; Tall
+*   RW37_7      0       0
+TEKGRENW        64      128 ; Wide
+*   RW37_8      0       0
+TEKGRENS        64      128 ; Square
```

Comment:
I don't understand these comments since they are all 64 x 128, but that's ok.

Reply:
Those comments describe the shape of the panels on those patches. Also, I had to use something other than a number for the suffix because `TEKGREN10` doesn't fit into 8 characters.